Marlin/ultralcd.cpp

```diff
@@ -113,6 +118,12 @@ uint8_t lcdDrawUpdate = LCDVIEW_CLEAR_CALL_REDRAW; // Set when the LCD needs to
   static void lcd_control_temperature_preheat_abs_settings_menu();
   static void lcd_control_motion_menu();
   static void lcd_control_volumetric_menu();
+  #ifdef DAC_STEPPER_CURRENT
+  static void dac_driver_commit();
```

Comment:
Point of style:
- Lines in-between `#if` and `#endif` should also be indented (2 spaces).